src/hci/gnomex/httpclient/SaveChromatogramsFromFiles.java

```diff
@@ -27,403 +27,406 @@
  */
 public class SaveChromatogramsFromFiles extends HttpClientBase {
 
-	private static int sleepInterval = 300000; // Number of milliseconds to sleep between running. defaults to 300 seconds (5 mins).
-
-	private boolean debug = false;
-	private String dropFilePath;
-	private String archiveFilePath;
-	private Integer idPlateWell;
-	private File currentFile;
-	private String filePath;
-	private String currentFD;
-	private StringBuffer errors;
-	private int numerrors;
-	private boolean noemail = false;
-	private String toAddress = "gnomex.support@hci.utah.edu;LabTech_Group@genetics.utah.edu"; // "LabTech_Group@genetics.utah.edu";
-	private String fromAddress = "GNomEx.Support@hci.utah.edu";
-
-	/**
-	 * @param args
-	 */
-	public static void main(String[] args) {
-
-		SaveChromatogramsFromFiles saveChromatograms = new SaveChromatogramsFromFiles(args);
-
-		while (true) {
-			try {
+private static int sleepInterval = 300000; // Number of milliseconds to sleep between running. defaults to 300 seconds (5 mins).
+
+private boolean debug = false;
+private String dropFilePath;
+private String archiveFilePath;
+private Integer idPlateWell;
+private File currentFile;
+private String filePath;
+private String currentFD;
+private StringBuffer errors;
+private int numerrors;
+private boolean noemail = false;
+private String toAddress = "Tim.Maness@hci.utah.edu"; // "LabTech_Group@genetics.utah.edu";
```

Comment:
It looks like you overwrote my change for the email address.  Was this intentional?